source/draw/Color.ooc

```diff
@@ -176,38 +176,72 @@ ColorBgra: cover {
10486, -10593, -10700, -10807, -10914, -11021, -11128, -11235, -11342, -11449, -11556, -11663, -11770, -11877, -11984, -12091, -12198, -12305, -12412, -12519, -12626, -12733, -12840, -12947, -13054, -13161, -13268, -13375, -13482, -13589, -13696, -13803, -13910, -14017, -14124, -14231, -14338, -14445, -14552, -14659, -14766, -14873, -14980, -15087, -15194, -15301, -15408, -15515, -15622, -15729, -15836, -15943, -16050, -16157, -16264, -16371, -16478, -16585, -16692, -16799, -16906, -17013, -17120, -17227, -17334, -17441, -17548, -17655, -17762, -17869, -17976, -18083, -18190, -18297, -18404, -18511, -18618, -18725, -18832, -18939, -19046, -19153, -19260, -19367, -19474, -19581, -19688, -19795, -19902, -20009, -20116, -20223, -20330, -20437, -20544, -20651, -20758, -20865, -20972, -21079, -21186, -21293, -21400, -21507, -21614, -21721, -21828, -21935, -22042, -22149, -22256, -22363, -22470, -22577, -22684, -22791, -22898, -23005, -23112, -23219, -23326, -23433, -23540, -23647, -23754, -23861, -23968, -24075, -24182, -24289, -24396, -24503, -24610, -24717, -24824, -24931, -25038, -25145, -25252, -25359, -25466, -25573, -25680, -25787, -25894, -26001, -26108, -26215, -26322, -26429, -26536, -26643, -26750, -26857, -26964, -27071, -27178, -27285, 0, -21, -42, -63, -84, -105, -126, -147, -168, -189, -210, -231, -252, -273, -294, -315, -336, -357, -378, -399, -420, -441, -462, -483, -504, -525, -546, -567, -588, -609, -630, -651, -672, -693, -714, -735, -756, -777, -798, -819, -840, -861, -882, -903, -924, -945, -966, -987, -1008, -1029, -1050, -1071, -1092, -1113, -1134, -1155, -1176, -1197, -1218, -1239, -1260, -1281, -1302, -1323, -1344, -1365, -1386, -1407, -1428, -1449, -1470, -1491, -1512, -1533, -1554, -1575, -1596, -1617, -1638, -1659, -1680, -1701, -1722, -1743, -1764, -1785, -1806, -1827, -1848, -1869, -1890, -1911, -1932, -1953, -1974, -1995, -2016, -2037, -2058, -2079, -2100, -2121, -2142, -2163, -2184, -2205, -2226, -2247, -2268, -2289, -2310, -2331, -2352, -2373, -2394, -2415, -2436, -2457, -2478, -2499, -2520, -2541, -2562, -2583, -2604, -2625, -2646, -2667, -2688, -2709, -2730, -2751, -2772, -2793, -2814, -2835, -2856, -2877, -2898, -2919, -2940, -2961, -2982, -3003, -3024, -3045, -3066, -3087, -3108, -3129, -3150, -3171, -3192, -3213, -3234, -3255, -3276, -3297, -3318, -3339, -3360, -3381, -3402, -3423, -3444, -3465, -3486, -3507, -3528, -3549, -3570, -3591, -3612, -3633, -3654, -3675, -3696, -3717, -3738, -3759, -3780, -3801, -3822, -3843, -3864, -3885, -3906, -3927, -3948, -3969, -3990, -4011, -4032, -4053, -4074, -4095, -4116, -4137, -4158, -4179, -4200, -4221, -4242, -4263, -4284, -4305, -4326, -4347, -4368, -4389, -4410, -4431, -4452, -4473, -4494, -4515, -4536, -4557, -4578, -4599, -4620, -4641, -4662, -4683, -4704, -4725, -4746, -4767, -4788, -4809, -4830, -4851, -4872, -4893, -4914, -4935, -4956, -4977, -4998, -5019, -5040, -5061, -5082, -5103, -5124, -5145, -5166, -5187, -5208, -5229, -5250, -5271, -5292, -5313, -5334, -5355 ]
```

Comment:
Maybe we can add the feature in rock some time. I used `#` to begin a line of C code so that precompiler statements then starts with `##`.